endoreg_db/management/commands/summarize_db_content.py

```
@@ -0,0 +1,189 @@
+                                if isinstance(min_val, datetime.datetime) and is_aware(min_val):
+                                    min_val = make_naive(min_val).strftime('%Y-%m-%d %H:%M:%S')
+                                elif isinstance(min_val, datetime.date):
+                                    min_val = min_val.strftime('%Y-%m-%d')
+                                
+                                if isinstance(max_val, datetime.datetime) and is_aware(max_val):
+                                    max_val = make_naive(max_val).strftime('%Y-%m-%d %H:%M:%S')
+                                elif isinstance(max_val, datetime.date):
+                                    max_val = max_val.strftime('%Y-%m-%d')
+
+                                self.stdout.write(f"  {field_obj.verbose_name.capitalize()} range: {min_val} to {max_val}")
+                                processed_date_field = True
```

Ruff (0.11.9)

Local variable `processed_date_field` is assigned to but never used

Remove assignment to unused variable `processed_date_field`

(F841)